src/taco-cli/cli/utils/cordovaWrapper.ts

```diff
@@ -188,6 +190,24 @@ class CordovaWrapper {
         options: { logLevel?: tacoUtility.InstallLogLevel, isSilent?: boolean, captureOutput?: boolean } = {}): Q.Promise<T | string> {
         return CordovaHelper.tryInvokeCordova<T | string>(apiFunction, () => CordovaWrapper.cli(processArgs(), options.captureOutput), options);
     }
+
+    /**
+     * Returns a path to the local Cordova command if ran from a taco project, otherwise it will return the
+     * global Cordova command.
+     */
+    private static getCordovaCommand(): Q.Promise < string > {
+        return CordovaHelper.tryInvokeCordova<string>((cordova: typeof Cordova): string => {
+        return (<{ packageTargetPath?: string }> cordova).packageTargetPath;
```

Comment:
Yes, we should read the package.json. 
In fact, now when I think about it, this calls for a different API, say lazyRun("packageName@1.0.0". since we are not really requiring here